docs/ReleaseNotes.txt

```diff
@@ -1,3 +1,113 @@
+=== v1.25.0 === (TBD October 2024)
```

Comment:
Should we mention the changes in standard library implementation and new `stdlib.isph`?